Guide.docc/Swift6Mode.md

```diff
@@ -41,24 +39,94 @@ let package = Package(
         // ...
     ],
     targets: [
-        // Uses the default tools language mode
+        // Uses the default tools language mode (6)
         .target(
             name: "FullyMigrated",
         ),
         // Still requires 5
         .target(
             name: "NotQuiteReadyYet",
             swiftSettings: [
-                .swiftLanguageVersion(.v5)
+                .swiftLanguageMode(.v5)
```

Comment:
Should take this change regardless - `swiftLanguageVersion` would result in a deprecation warning.

Reply:
I see, good catch but tbh the entire PR looks good to me hmm